Paws4Me/Paws4Me/Controllers/Logic/AllPetDetailViewController.swift

```diff
@@ -129,19 +129,19 @@ class ViewController: UIViewController, UISearchBarDelegate {
                let indexRow = getIndexPetSelected()
                guard let pageItem = adoptPetObject?.page?[indexRow] else { return }
                if let pagePetName = pageItem.name {
-                   destination.namePet = pagePetName
+                   destination.setNamePet(name: pagePetName)
```

Comment:
So over here it would be better to just pass the whole `pageItem` 😄

Reply:
Will do so 👍🏻